bitcash/wallet.py

```diff
@@ -14,19 +14,24 @@
     )
 
 
-def wif_to_key(wif):
-    private_key_bytes, compressed, version = wif_to_bytes(wif)
-
+def wif_to_key(wif, regtest=False):
+    private_key_bytes, compressed, version = wif_to_bytes(wif, regtest)
+    
     if version == 'main':
         if compressed:
             return PrivateKey.from_bytes(private_key_bytes)
         else:
             return PrivateKey(wif)
-    else:
+    elif version == 'test':
         if compressed:
             return PrivateKeyTestnet.from_bytes(private_key_bytes)
         else:
             return PrivateKeyTestnet(wif)
+    else: # Regtest
```

Comment:
I'm being extremely picky here, but can we have two white spaces before the comment as specified by [PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments)? 😅

Reply:
Done [https://github.com/pybitcash/bitcash/pull/82/commits/beb7598fe036e8be52a417024da4debcca976585](https://github.com/pybitcash/bitcash/pull/82/commits/beb7598fe036e8be52a417024da4debcca976585)